src/modules/market/selectors/market.js

```diff
@@ -291,7 +291,7 @@ export function assembleMarket(
 
       market.tags = (market.tags || []).map((tag) => {
         const obj = {
-          name: tag && tag.toString().toLowerCase().trim(),
+          name: tag && tag.toString().trim(),
```

Comment:
@stephensprinkle Dumb as it sounds, I don't think we even want to trim the tags, since trimmed tags will not match untrimmed tags on-contract.  I can't think of a valid use-case for untrimmed tags, so I'm making it so `formatTag` in augur.js trims tags (during `createMarket`).  However it's always possible some enthusiast will send an `     untrimmed tag rofl   ` to the Serpent API directly and that probably should cause his market to be completely invisible to the UI!